firefox_ios/views/firefox_ios_feature_usage_events.view.lkml

```diff
@@ -0,0 +1,410 @@
+include: "//looker-hub/firefox_ios/views/feature_usage_events.view.lkml"
+
+view: +feature_usage_events {
+
+  extends: [feature_usage_events]
+
+  measure: sum_app_icon_new_private_tab_tapped {
```

Comment:
are these dimensions on the original view? Do you need the dimensions as well, or should they be hidden?